shard.yml

```diff
@@ -16,3 +16,5 @@ dependencies:
     version: ~> 0.1.1
 
 crystal: 0.27.1
```

Comment:
Prob could bump this too

Reply:
This should contain the **minimum** Crystal version this shard is working on, not the _current_ Crystal version, and anyway that's not in a scope of this PR.

Reply:
Agreed that it's not the scope of this PR, but `crystal` does not specify the "minimum version". It's rather the ["last known Crystal version that is capable to compile the Shard"](https://github.com/crystal-lang/shards/blob/master/SPEC.md#crystal).

Reply:
@straight-shoota Hmm, you're right according to the SPEC, although I still believe it should be the _minimum_ version, otherwise what's the point? Should we bump it for _every_ shard, _every time_ there's new Crystal version released?

Reply:
That's truly questionable. Until there is a 1.0 release with compatibility guarantees, it doesn't really matter either way. But yeah, in the long term, it would probably be better to mean the minimum required version.